test/utils/helpers.ts

```diff
@@ -210,6 +206,10 @@ export function getCorrespondingCollateralCurrenciesForTests(
     }
 }
 
+export function getWrappedCurrencyForTest(api: ApiPromise): WrappedCurrency {
+    const currencyId = api.consts.currency.getWrappedCurrencyId;
+    return tokenSymbolToCurrency(currencyId.asToken);
+}
```

Comment:
Already declared as a getter on `InterBtcApi`

Reply:
Right, I'll reuse that one then :+1: